### PR TITLE
Added update site for Gut Analysis Toolbox

### PR DESCRIPTION
**Pull request by pr4deepr**
Hi
  I would like to add my software Gut Analysis Toolbox to the list of update sites please. 
Thank you.

Cheers
Pradeep

### CODE REVIEW

**Review by ctrueden — CHANGES_REQUESTED** [review]
@pr4deepr Thanks!

There is an error with a missing leading square bracket in your Markdown description. Could you please fix that?

Also, could you please create a page on imagej.net for your plugin(s)? It's up to you whether you populate it fully, or just make stubs linking off to your GitHub wiki docs (although I personally wish more of the community would unify around the ImageJ wiki so that users can actually find stuff more easily).

File: sites.yml [hunk 1]
```diff
@@ -834,6 +834,17 @@ sites:
     maintainers:
       - "[Robert Erger](mailto:rerger@leomail.tamuc.edu)"
 
+  - name: "Gut Analysis Toolbox"
+    id: "GAT"
+    url: "https://sites.imagej.net/GutAnalysisToolbox/"
+    description: >-
+      Gut Analysis Toolbox](https://github.com/pr4deepr/GutAnalysisToolbox/wiki) enables semi-automated analysis of
```
[the remark this answers is not in the document]
Comment (pr4deepr) [reply sites.yml]: Created a pull request:
https://github.com/imagej/list-of-update-sites/pull/107/commits/376d9bac05fe12d8fc0b7fe44f2b5af06731e737

File: sites.yml [hunk 2]
```diff
@@ -834,6 +834,17 @@ sites:
...
+  - name: "Gut Analysis Toolbox"
+    id: "GAT"
+    url: "https://sites.imagej.net/GutAnalysisToolbox/"
+    description: >-
+      Gut Analysis Toolbox](https://github.com/pr4deepr/GutAnalysisToolbox/wiki) enables semi-automated analysis of 
+      the neuronal and glial distribution within the gut wall in 2D. It uses a combination of FIJI macros, StarDist 
+      and deepImageJ for automated segmentation. The primary focus is on characterising the enteric nervous system.
+      The aim is to expand the analysis to other celltypes within the gut and enable 3D analysis.
+    maintainers:
+      - "[Pradeep Rajasekhar](mailto:pradeep.rajasekhar@gmail.com)"
```
[the remark this answers is not in the document]
Comment (pr4deepr) [reply sites.yml]: Thanks
Created a pull request..
https://github.com/imagej/imagej.github.io/pull/227
I wasn't aware of creating pages in imagej.net.. I'd prefer that than github as its easier to discover..

**Comment by ctrueden** [comment]
@pr4deepr Great, thanks!

**Pull request merged**